src/core/plugins/core-items/server/src/registerItems.ts

```diff
@@ -1,11 +1,17 @@
 import { ItemFactory } from '../../../../server/systems/item';
 import { ITEM_TYPE } from '../../../../shared/enums/itemTypes';
 import { getWeaponList } from '../../../../shared/information/weaponList';
-import { items } from './items';
-
+import { drinks } from './items/drinks';
+import { food } from './items/food';
+import { utility } from './items/utility';
 export class RegisterItems {
     static async init() {
-        const itemsToAdd = [...items];
+        const itemsToAdd = [
```

Comment:
Would create in `index.ts` file inside of the items folder and then

```ts
export * from './food';
export * from './drinks';
export * from './drinks'
```

Then your itemsToAdd just looks
```ts
const itemsToAdd = [...items];
```